bin/checkVersion.js

```diff
@@ -0,0 +1,48 @@
+/*eslint no-console: "off"*/
```

Comment:
I think we could set this for the whole `bin` directory (https://github.com/codefordenver/members/blob/master/bin/.eslintrc), but this is fine for now.